Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,16 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
+// Global static objects. Initialized before fuzzing starts.
+static Serializer *serializer;
```

Comment:
Note that making a global variable `static` and placing it into an anonymous namespace both achieve pretty much the same goal (containing the object within a single translation unit to prevent accidental violation of One Definition Rule). Anonymous namespaces are generally considered to be a [superior alternative](https://stackoverflow.com/questions/4977252/why-an-unnamed-namespace-is-a-superior-alternative-to-static) in C++, so I'd advise to remove `static` here.